easysnmp/interface.c

```diff
@@ -84,6 +84,10 @@ static void py_log_msg(int log_level, char *printf_fmt, ...);
 
 enum { INFO, WARNING, ERROR, DEBUG, EXCEPTION };
 
+static PyObject *easysnmp_Import = NULL;
```

Comment:
Hey mate, can we switch the I to lowercase here please? (and similarly across the other variables?) i.e. `easysnmp_import`. :smile: